src/services/rate/client/exchangeapi.rate.service.ts

```diff
@@ -0,0 +1,19 @@
+import {Fawazahmed0ExchangeRateData, IRateService} from "../rate.service.interface";
+import axios from "axios";
+import logger from "../../../helpers/logger";
+import {EXCHANGE_API_URL} from "../../../config/rate.api.const";
+
+export class ExchangeAPIRateService implements IRateService{
+
+    async getExchangeRate(): Promise<number> {
+        const response = await axios.get<Fawazahmed0ExchangeRateData>(EXCHANGE_API_URL);
+        logger.info(`cdn.jsdelivr.net - Response: ${JSON.stringify(response.data)}`);
```

Comment:
If the requirements for the logging format change in the future, you will have to make changes to each client file. Now, there are not many of them, but in a case with, for example, 100 clients, it will be a pain. You should use Decorator pattern here to decorate logging